src/app/data/consts.js

```diff
@@ -0,0 +1,33 @@
+class Consts {
```

Comment:
I am a newbie in OOP, so it's a question for @vieraboschkova .
Maybe coping with constant values is easier  without using classes?

Reply:
Well, I must say I never saw it done through a class :) If you define a js file with constants, you could import just the ones that you really need. I guess in your application size it doesnt really matter, but it could present performance issues if it grew, perhaps you could do further research about that.